Gemfile

```diff
@@ -132,6 +132,8 @@ source "https://rails-assets.org" do
   gem "rails-assets-numeral",                         "~>1.5.3"
   gem "rails-assets-slickgrid",                       "~>2.1.0"
   gem "rails-assets-jqueryui",                        "~>1.9.2"
+  gem "rails-assets-angular-patternfly",              "=2.4.0"
+  gem "rails-assets-angular-animate",                 "=1.4.5"
```

Comment:
Please keep the version string identical to `rails-assets-angular` here (which would be `~>1.4.3`), we've already hit some problems with inconsistent angular module versions. And maybe the same goes for angular-patternfly - patternfly-sass has `~>2.4.0`.